### ISSUE TITLE
Show the hard block volume fraction in corfunc

### ISSUE THREAD

**Issue by lucas-wilkins** [SasView/sasview#2110]
Migrated from #1502

"compute & display the "Hard Block Volume Fraction"

What is the hard block volume fraction @smk78

**Comment by lucas-wilkins** [comment]
> What is the hard block volume fraction @smk78

nudge @smk78

**Comment by lucas-wilkins** [comment]
@smk78 ??

**Comment by smk78** [comment]
Right... this turned out to be a head slapper! I remembered reading something somewhere but it's taken me time to find it again (for reference, footnote 80 on p145 of Stribeck's book!). The hard block volume fraction is nothing more than an alternative name for the local crystallinity! But, it is a better, more generic, name! The reason being that if you use a word like 'crystallinity' what interpretation do you take away if you know your sample is not actually crystalline? Correlation function analysis is not limited to crystalline samples, it's just that the proponents that developed the field mostly worked with crystalline polymers. So, to summarise, this is just a change of wording on the Corfunc GUI.

**Comment by smk78** [comment]
However, I think there is another issue about verifying exactly what values are being displayed for these parameters? It might be an idea to reconsider the parameter names as that is done... Probably best if we do that jointly @lucas-wilkins .

**Comment by lucas-wilkins** [comment]
> It might be an idea to reconsider the parameter names as that is done... Probably best if we do that jointly @lucas-wilkins .

OK, I'm back on the 15th. I also notice that the output fields lack tool tips, this might be a place to mention that there are different names.
